### DISCUSSION TITLE
Welcome to Diablo Discussions!

### DISCUSSION THREAD

**Discussion by AnLoMinus** [AnLoMinus/Diablo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gijhu** [comment]
How do I open the tool? Does it work with Python?

**Reply by AnLoMinus** [reply]
hi ! 👋 
its bash tool. 

try in terminal
[ ./diablo.sh ] or [ sudo chmod +x diablo.sh; ./diablo.sh ]

tell me if it helped
